rbac/management/role/model.py

```diff
@@ -83,14 +83,7 @@ def permission_application(self):
 
     def split_permission(self):
         """Split the permission."""
-        return self.permission.split(":")
-
-    def save(self, *args, **kwargs):
```

Comment:
Same with this, pulling this out and not updating it to store the `permission` field will leave the `permission` field with the default value of `*:*:*`.

Reply:
yea, this permission field will not be used any more, we are going to delete this permission field, and recreate it in the following PR